src/openvic-simulation/scripts/Condition.cpp

```diff
@@ -505,9 +505,8 @@ callback_t<std::string_view> ConditionManager::expect_parse_identifier(
 		EXPECT_CALL(BUILDING, building_type, definition_manager.get_economy_manager().get_building_type_manager(), "FACTORY");
 		EXPECT_CALL(CASUS_BELLI, wargoal_type, definition_manager.get_military_manager().get_wargoal_type_manager());
 		EXPECT_CALL(GOVERNMENT_TYPE, government_type, definition_manager.get_politics_manager().get_government_type_manager());
-		EXPECT_CALL(MODIFIER, event_modifier, definition_manager.get_modifier_manager());
-		EXPECT_CALL(MODIFIER, triggered_modifier, definition_manager.get_modifier_manager());
-		EXPECT_CALL(MODIFIER, static_modifier, definition_manager.get_modifier_manager());
```

Comment:
So are we not able to check if a country or province has a static modifier (other than those you've treated as event modifiers)?

Reply:
Correct. Victoria 2 only treats certain static modifiers as event modifiers.
I confirmed bad_debter and in_bankrupcy. If we find others, we can add them to the code.